src/services/migration/ProjectAuthXmlMigration.php

```diff
@@ -42,7 +42,7 @@ public function migrate(): void {
         $newContent = \str_replace('https://phar.io/phive-auth', 'https://phar.io/auth', $oldContent);
         $new->putContent($newContent);
 
-        $this->output->writeText('Migration of authentication configuration almost finish.');
+        $this->output->writeText('Migration of authentication configuration almost finish. ');
```

Comment:
and last but not least..

Reply:
At least I'm constant in what I'm write 😃